automation/helpers/json_responses.py

```diff
@@ -44,7 +44,7 @@
             "enum": [
               "fence",
               "dcf-fence",
-              "memcache",
+              "cache",
```

Comment:
Was the error messaging informative?

Reply:
Yeah, though I'm only getting them through Greg since I can't see the jenkins.

```
ERROR: test_status (api_test.PublicApiTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/app/automation/tests/api_test.py", line 24, in test_status
    validate(instance=response_json_dict, schema=json_schema_test_status)
  File "/app/lib/jsonschema/validators.py", line 934, in validate
    raise error
ValidationError: u'cache' is not one of ['fence', 'dcf-fence', 'memcache', 'datastore', 'sam']
Failed validating 'enum' in schema['properties']['subsystems']['items']['properties']['subsystem']:
    {'enum': ['fence', 'dcf-fence', 'memcache', 'datastore', 'sam'],
     'type': 'string'}
On instance['subsystems'][2]['subsystem']:
    u'cache'
```